doc/book/box/atomic/txn_mode_mvcc.rst

```diff
@@ -69,13 +69,13 @@ Transaction manager options
 
 The transaction manager has four options for the transaction isolation level that you can set in ``box-cfg``:
 
-*   ``default``.
+*   ``default``
 
-*   ``best-effort``.
+*   ``best-effort``
 
-*   ``read-committed``.
+*   ``read-committed``
 
-*   ``read-confirmed``.
+*   ``read-confirmed``
 
```

Comment:
Let's describe a little bit all the options here, not just the best-effort. Otherwise, it looks like you'll have to guess the rest or read about it on the previous pages.